src/frontend/Sider/Behandling/Vilkårvurdering/EndreVilkår.tsx

```diff
@@ -218,7 +228,19 @@ export const EndreVilkår: FC<EndreVilkårProps> = (props) => {
     const EndrePerioder = (
         <HStack gap="4" align="start">
             <FeilmeldingMaksBredde $maxWidth={152}>
-                {skalBrukeMånedÅrVelger ? (
+                {erMidlertidigOvernatting ? (
```

Comment:
Tror ikke dette skalerer så veldig godt 😅 Allerede litt vondt med to nøstede ternary-operatorer

Reply:
Enig! Den ene dato-velgeren er for at Tone++ skal teste ut om vi kan behandle mer rett hvis SB ikke får mulighet til å velge dag, men kun måned. Tenker vi kan rydde litt opp her når vi har landet om vi vil bruke vanlig dato-velger eller dropdown dato-velger.